README.md

```diff
@@ -1,3 +1,141 @@
 # QA Decontextualization
 
-Code and data coming soon!
+See `experiments` for description of how to run experiments investigating this method.
+
+## Set Up
+
+```bash
+conda create  -n qa_decontext python=3.9
+conda activate qa_decontext
+pip install -e .
+```
+
+## Quick Start
+
+1. Set your OpenAI API key
+```bash
+export OPENAI_API_KEY='yourkey'
+```
+
+2. Decontextualize
+
+To decontextualize a snippet using some context, you can pass both the snippet and context to the decontextualization function.
+```python
+from qa_decontext import decontextualize
+
+context = """\
```

Comment:
Note that the context should include the title, abstract, and context.

Also could do string and start and end of index.

Could also allow for input of a dict (eg paper snippet).